### ISSUE TITLE
we need one click the tab to open the tab

### ISSUE THREAD

**Issue by jiqimaogou** [GarinZ/link-map#89]
hello can we open the tab just one click instead of double click?

**Comment by AAraKKe** [comment]
I just discovered this extension, and it is probably the best solution to sideberry in Chrome. However, the one click feature missing is driving me nuts xD it would be really nice to be able to just click on the tab to open it. I am really considering paying for the pro version of it but the friction of the double click has me often just not using it :(

**Comment by PowerWeb5** [comment]
I agree, one click to open tab is the key limitation keeping me from using Link Map vs. Tabs Outliner (despite it not being updated since 2021).  Please considering implementing this. I too would pay if had that.
